secant.xcodeproj/project.pbxproj

```diff
@@ -599,6 +641,7 @@
 				0D4E7A1926B364180058B01E /* secantTests */,
 				0D4E7A2426B364180058B01E /* secantUITests */,
 				0D4E7A0626B364170058B01E /* Products */,
+				9EB863872922C394003D0F8B /* Recovered References */,
```

Comment:
uhhmm this is always in indication that something can be going on with the project file.

Reply:
resolved